src/openrct2/scripting/bindings/world/ScTileElement.cpp

```diff
@@ -406,37 +406,40 @@ namespace OpenRCT2::Scripting
         }
         return DukValue::take_from_stack(ctx);
     }
-    void ScTileElement::sequence_set(uint8_t value)
+    void ScTileElement::sequence_set(const DukValue& value)
```

Comment:
Why not change this just to `uint16_t`? We don't handle any other type of value.

Reply:
For `TrackElement`s that are Mazes, the `sequence` is `null`, which means that `element.sequence = element.sequence` would throw an error if the parameter is of type `uint16_t`.